### ISSUE TITLE
ci(windows): `build x86_64-windows` takes a long time.

### ISSUE THREAD

**Issue by nabetti1720** [awslabs/llrt#699]
In `build (windows-latest, windows, x86_64, nightly-x86_64-pc-windows-gnu) / build x86_64-windows`, even though the test is completed, it does not proceed to the next STEP for a long time.

<img width="757" alt="image" src="https://github.com/user-attachments/assets/34a6a37e-d3f2-480b-9a0f-6d4014bd15ad">

I have no idea what I'm waiting for...

**Comment by ahaoboy** [comment]
Maybe need to add exit function
```
  shutdown() {
    this.server?.close();

  //     process.exit(0)
  }
```

**Comment by richarddavison** [comment]
I think there is a race condition here doesn't close the server or child processes properly. We need to investigate.

**Comment by nabetti1720** [comment]
This has been fixed in #721 so I will close it.
